base_classes/NXdata.nxdl.xml

```diff
@@ -309,6 +309,20 @@
 
 		</doc>
 	</attribute>
+	<attribute name="target">
```

Comment:
Why have this attribute attached to the group in addition to the one for `DATA`?